pages/phases/certificate.md

```diff
@@ -24,23 +24,30 @@ roles: [hr, hiring-mgr]
+During resume review and interview assessments, applicants can be removed from consideration if SMEs determine they're not qualified based on the required competencies and proficiencies. As HR, you should apply Veterans' preference only to applicants who achieve an overall passing score, then issue the certificate of eligibles of the "Best Qualified" category to the hiring manager.
 
 ## Post-Certificate Interviews
 
-After the hiring certificate is issued, agencies can use the same procedures they always have. Hiring managers may conduct additional interviews with applicants from the certificate, or choose instead to rely on the interview transcript notes to determine whether they'll make offers to applicants on the certificate.
+Hiring managers may conduct additional interviews with applicants from the certificate, or choose instead to rely on the interview transcript notes to determine whether they'll make selections from the certificate.
 
-To pass over a veteran, hiring managers must meet the requirements of 5 U.S.C. § 3318 and submit a pass over request. For more information, see the "[Vet Guide for HR Professionals](https://www.opm.gov/policy-data-oversight/veterans-services/vet-guide-for-hr-professionals/)" on the *OPM Website*.
+To pass over a veteran, hiring managers must meet the requirements of 5 U.S.C. § 3318 and submit a pass over request. Due to the rigorousness of this process, it's unlikely you'll be able to pass over a vet for lack of qualifications. However, a pass over request is valid if the applicant isn't a good fit for your office based on reference checks or interviews with hiring managers. For more information, see the "[Vet Guide for HR Professionals](https://www.opm.gov/policy-data-oversight/veterans-services/vet-guide-for-hr-professionals/)" on the *OPM Website*.
 
 ## Selecting Qualified Applicants from a Certificate
 
-When hiring with this process, there should be fewer applicants who make it through qualifications and onto the certificate, with more selections off of the delegated examining certificates. Ideally, the extraordinary SME review process that qualifies applicants will allow applicants on the certificate to receive offers for positions at an agency. If there are more qualified applicants on a certificate than available billets at an agency, that agency may choose to share the certificate with other groups or agencies.
+When hiring with this process, there should be *fewer* applicants who make it to onto the certificate based on their qualifications and *more* selections off of the delegated examining certificates. Ideally, the extraordinary SME review process that qualifies applicants will allow applicants on the certificate to receive offers for positions at an agency. If there are more qualified applicants on a certificate than available billets at an agency, that agency may choose to share the certificate with other groups or agencies.
```

Comment:
instead of "that qualifiles applicants" say "confirms technical qualifications are met"

Reply:
instead of "will allow applicants" to "will result in all qualified applicants to receive offers..."